src/passes/InstrumentMemory.cpp

```diff
@@ -309,6 +351,20 @@ struct InstrumentMemory : public WalkerPass<PostWalker<InstrumentMemory>> {
     curr->addFunction(std::move(import));
   }
 };
+struct InstrumentMemory : Pass {
```

Comment:
```suggestion

struct InstrumentMemory : Pass {
```

Reply:
Surprised the linting doesn't warn about this... anyhow, we like to have spacing after classes, as a convention.